src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -387,6 +387,7 @@ class DevTask extends AbstractFeatureTask {
                     true /* useBuildRecompile */, true /* gradle */, pollingTest, container, containerfile, containerBuildContext, containerRunOpts, containerBuildTimeout, skipDefaultPorts,
                     null /* compileOptions not needed since useBuildRecompile is true */, keepTempContainerfile, mavenCacheLocation, projectModuleList /* multi module upstream projects */,
                     projectModuleList.size() > 0 /* recompileDependencies as true for multi module */, packagingType, buildFile, parentBuildGradle /* parent build files */, generateFeatures, null /* compileArtifactPaths */, null /* testArtifactPaths */, webResourceDirs /* webResources */
+                    ,Collections.emptyMap() /*ignored for gradle*/
```

Comment:
This PR would be unnecessary if you provided a constructor that did not take this new Map since it is not needed for Gradle. That constructor could then just turn around and call the other one passing an empty Map, but in ci.common instead of ci.gradle.